.github/ISSUE_TEMPLATE/01_bugs.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "\U0001F6A8 Bug Report"
+about: Did you come across a bug or unexpected behaviour differing from the docs?
+labels: bug
+
+---
+<!--
+Thanks for reporting a bug 🙌 ❤️
+
+Before opening a new issue, please make sure that we do not have any duplicates already open. You can ensure this by searching the issue list for this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead.
+
+Also, be sure to check our documentation first: https://github.com/Digitaler-Impfnachweis/documentation
+-->
+
+## Avoid duplicates
+* [ ] Bug is not mentioned in the [FAQ](https://digitaler-impfnachweis-app.de/faq/)
+* [ ] Bug is not already reported in another issue
+
+## Technical details
+
+- Device name:
+- iOS version:
```

Comment:
Maybe we should use `OS version` so we can use the exact template for Android

Reply:
Will change ASAP. Should I prepare a PR for the Android repo?

Reply:
Done in https://github.com/Digitaler-Impfnachweis/covpass-ios/pull/27/commits/2150c425936c7b8a8844854252438011e3273bc0.